packages/react-dom/src/server/ReactDOMStringRenderer.js

```diff
@@ -12,7 +14,7 @@ import ReactPartialRenderer from './ReactPartialRenderer';
  * server.
  * See https://reactjs.org/docs/react-dom-server.html#rendertostring
  */
-export function renderToString(element) {
+export function renderToString(element: Element) {
```

Comment:
This is a DOM element. It's not related to React elements (which are the inputs to this function).

Reply:
Oh okay. So flow type here should be `any`. Right?

Reply:
Kind of, but I don’t see how this helps us in general. It’s entry point so its type doesn’t matter in the internal flow check.